### ISSUE TITLE
Share to Subconscious

### ISSUE THREAD

**Issue by bfollington** [subconsciousnetwork/subconscious#351]
Came up while dogfooding the app. 

Sharing a URL or a selected text passage to Subconscious via the iOS share sheet would be extremely useful. Similar to how Readwise works.

For a plain URL we can just copy it into the content. For a selected text passage perhaps we could use a format like:

```
> The selected text you chose to share 

Source: URL / source application name
```

Not sure how to generate a useful title, might be a case for `YYYY-MM-DD` plus a sanitised version of the source? 

It might better to prompt the user for the title during the share interaction?

Could also take inspiration from https://museapp.com/ and have an "inbox" you have to process in-app, but I fear this creates too much overhead.

**Comment by bmann** [comment]
Yeah less inboxes. Over time people could eg auto tag with [[inbox]] if they like, but default can just save. 

And yes, +1 to having a normalized “daily log” format in such a way that individual items can easily be seen when they have been posted.
